### ISSUE TITLE
Cannot add guests to event on calendar shared with me

### ISSUE THREAD

**Issue by vitoreiji** [tutao/tutanota#4053]
**To Reproduce**
1. Alice shares calendar with Bob, with permission level *Read and write*
3. Bob accepts invitation
4. Bob creates event on shared calendar
5. Bob tries to add guests to event, but can't

![image](https://user-images.githubusercontent.com/197261/164410709-b34305a0-eea0-4c78-b8c1-70635fa3e29f.png)
Clicking on "GUESTS" should display a new row allowing Bob to *Add a guest*. 

**Expected behavior**
Bob is able to add guests to the event.
![image](https://user-images.githubusercontent.com/197261/164411457-f64b2a9d-a3a1-421d-aee4-204abebf05d4.png)

**Additional context**
This is also the case with permission level *Write and manage sharing*.

Alice is able to add guests as expected.

**Comment by charlag** [comment]
Is this a duplicate of #3762 ?

**Comment by vitoreiji** [comment]
I don't think so. That one is more about some warning, this one is about being able to add guests.

**Comment by charlag** [comment]
The reason there is that the type of the event is detected incorrectly and I think the reason is the same

**Comment by wrdhub** [comment]
#5785 is a bit related here, the reason we don't allow this right now is because updates cannot come from someone who is not the organizer.

**Comment by charlag** [comment]
This is currently by design because of restrictions on who can send calendar event updates. We will address it in the future one way or another.

**Comment by kibhub** [comment]
Sorry, I think this issue should be reopened in order to keep track of it. The issue comes up quite a bit in support.
